openquake/hazardlib/contexts.py

```diff
@@ -175,6 +172,11 @@ def make_contexts(self, sites, rupture):
             Instance of
             :class:`openquake.hazardlib.source.rupture.BaseRupture`
 
+        :param reqv_point:
+            True if reqv is set and source is a point or area source.
+            If True, rjb and rrup are replaced in the distances context
+            by equivalent values.
+
```

Comment:
It this new boolean parameter needed? I mean, we have already  the `self.reqv` boolean, can't we use that and only that?

Reply:
I added the boolean as a way to distinguish between point and area sources (for which the equivalent distance correction should be applied) and fault sources (for which the equivalent distance correction should NOT be applied, even when self.reqv is set). 

Point and area sources are distinguished by the line: if hasattr(src, 'location') or hasattr(src, 'area_discretization')

I'm open if you know a better way of doing this.

Reply:
Point sources and area sources produces PlanarSurfaces, so you can remove the flag and add a check like

```python
if self.reqv and isinstance(rupture, PlanarSurface)
```
```